src/main/views/case-standards-list.njk

```diff
@@ -0,0 +1,104 @@
+        {% if bill %}
+          <P>{{ importantInformationP4 }}</P>
+          <P>{{ importantInformationP5 | replace("EMAIL", venueEmail) }}</P>
+          <P>{{ importantInformationP6 }}</P>
+        {% endif %}
+      </div>
+    </details>
+
+    <div class="parent-box overflow-table">
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">{{ hearingDate }}</th>
+          <th scope="col" class="govuk-table__header">{{ caseName }}</th>
+          <th scope="col" class="govuk-table__header">{{ duration }} </th>
+          <th scope="col" class="govuk-table__header">{{ hearingType }}</th>
+          <th scope="col" class="govuk-table__header">{{ venue }}</th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        {% for courtCase in listData %}
+          {% set duration = '' %}
+          {% if courtCase['durationAsDays'] !== 0 %}
+            {% if courtCase['durationAsDays']  > 1 %}
+              {% set duration = courtCase['durationAsDays'] + ' ' + days %}
+            {% else %}
+              {% set duration = courtCase['durationAsDays'] + ' ' + day %}
+            {% endif %}
+          {% else %}
+            {% set durationAsHours = 0 %}
+            {% if courtCase['durationAsHours'] !== 0 %}
+              {% if courtCase['durationAsHours'] > 1 %}
+                {% set durationAsHours = courtCase['durationAsHours'] + ' ' + hours %}
+                {% elif courtCase['durationAsHours'] === 1 %}
+                {% set durationAsHours = courtCase['durationAsHours'] + ' ' + hour %}
+              {% endif %}
+            {% endif %}
+            {% set durationAsMinutes = 0 %}
+            {% if courtCase['durationAsMinutes'] !== 0 %}
+              {% if courtCase['durationAsMinutes'] > 1 %}
+                {% set durationAsMinutes = courtCase['durationAsMinutes'] + ' ' + mins %}
+                {% elif courtCase['durationAsMinutes'] === 1 %}
+                {% set durationAsMinutes = courtCase['durationAsMinutes'] + ' ' + min %}
+              {% endif %}
+            {% endif %}
+            {% if durationAsHours === 0 and durationAsMinutes === 0 %}
+              {% set duration = '' %}
+              {% elif durationAsHours !== 0 and durationAsMinutes !== 0 %}
+              {% set duration = durationAsHours + ' ' + durationAsMinutes %}
+              {% elif durationAsHours === 0 %}
+              {% set duration = durationAsMinutes %}
+              {% elif durationAsMinutes === 0 %}
+              {% set duration = durationAsHours %}
+            {% endif %}
+          {% endif %}
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">{{ courtCase['hearingDate'] }}</td>
+            <td class="govuk-table__cell">{{ courtCase['caseName'] }}</td>
+            <td class="govuk-table__cell">{{ duration }} {{ courtCase['caseSequenceIndicator'] }}</td>
```

Comment:
I have seen in other lists, the caseSequenceIndicator hasn't got any square brackets in the data, and the square brackets are added in the nunjucks file. Are you sure it is different for the care standards list?

Reply:
caseSequenceIndicator on frontend will be without brackets. Brackets must be part of Json. We should receive caseSequenceIndicator with brackets in json i.e. [1 of 2].